kotlinlib/src/mill/kotlinlib/ksp/KspModule.scala

```diff
@@ -76,16 +76,61 @@ trait KspModule extends KotlinModule { outer =>
   private val kspPluginId: String =
     "com.google.devtools.ksp.symbol-processing"
 
+  /**
+   * The KSP ap classpath
+   *
+   * For more info go to [[https://kotlinlang.org/docs/ksp-command-line.html]]
+   */
+  def kspApClasspath: T[Seq[PathRef]] = Task {
+    kotlinSymbolProcessorsResolved()
+  }
+
+  /**
+   * The sources for being used in KSP, in case
+   * the user wants to separate KSP specific sources
+   * from others. Defaults to [[sources]] (i.e. no splitting)
+   */
+  def kspSources: T[Seq[PathRef]] = Task {
+    sources()
+  }
+
+  /**
+   * The classpath when running Kotlin Symbol processing
+   *
+   * For more info go to [[https://kotlinlang.org/docs/ksp-command-line.html]]
+   */
+  def kspClasspath: T[Seq[PathRef]] = Task {
+    super.compileClasspath()
+  }
+
+  override def kotlinLanguageVersion: T[String] = "1.9"
```

Comment:
Why is this hard-coded here? If we only support "1.9", we should check it where we expect it to be "1.9". But silently overriding a possibly already configured values (via a trait mixed in before `KspModule` is bad style and hard to debug for users).

Reply:
the default is empty, and 1.9 is one that works with KSP . If it's not provided, KSP stops working

Reply:
I think I can just do, kspKotlinLanguageVersion

Reply:
I changed the defaults in the meantime. https://github.com/com-lihaoyi/mill/pull/4890

Reply:
ok, I tried it, but still doesn't seem to work without 1.9 source compatibility